lib/portfinder.js

```diff
@@ -135,11 +134,23 @@ exports.getPort = function (options, callback) {
     if(options.startPort < 0) {
       throw Error('Provided options.startPort(' + options.startPort + ') is less than 0, which are cannot be bound.');
     }
-    if(options.stopPort < options.startPort) {
-      throw Error('Provided options.stopPort(' + options.stopPort + 'is less than options.startPort (' + options.startPort + ')');
+  }
+
+  if (options.startPort > _highestPort) {
+    throw Error('Provided options.startPort(' + options.startPort + ') is higher than the maximum available port (' + _highestPort + ').');
+  }
+
+  if (options.startPort > exports.highestPort) {
+    console.warn('Provided options.startPort(' + options.startPort + ') is higher than the recommended stopPort (' + exports.stopPort + '). This may cause issues on some systems');
```

Comment:
`exports.stopPort` is `undefined`:

> Provided options.startPort(40001) is higher than the recommended stopPort (undefined). This may cause issues on some systems

It should be `export.highestPort`, isn't it?

Reply:
Oops - yes, good catch.